shared/SharedConfig.ts

```diff
@@ -0,0 +1,9 @@
+export const sharedConfig = {
+    FREE_TIER_PRIVATE_VOTER_LIMIT: 2, // making really low at first so it's easy to test
```

Comment:
Can we increase this to 20? Would like to be able to test with more voters in private elections. We can lower it locally for testing.

Reply:
This is still only applied in the front end right? Where does the backend use this to reject the request?

Reply:
You're right I messed up the rebase and lost the most recent commits. Should be fixed now

(more specifically I did the rebase on my desktop, but the latest version of the branch was from my laptop, so I overwrite the laptop changes with the force push)